src/util.c

```diff
@@ -627,6 +627,93 @@ int d2string(char *buf, size_t len, double value) {
     return len;
 }
 
+/* Convert a double into a string with 'fractional_digits' digits after the dot precision.
```

Comment:
Make sure to document the limitations (I.e. Overflow concerns) 
And document all inputs, outputs and error handling

Reply:
@oranagra can you check if the current comment is OK?